internal/gitprovider/gitprovider.go

```diff
@@ -151,3 +156,11 @@ func (f *Fake) ListPullRequests(
 ) ([]PullRequest, error) {
 	return f.ListPullRequestsFn(ctx, opts)
 }
+
+// GetCommitURL implements gitprovider.Interface.
+func (f *Fake) GetCommitURL(
+	repoURL string,
+	sha string,
+) (string, error) {
```

Comment:
```suggestion
func (f *Fake) GetCommitURL(repoURL string, sha string) (string, error) {
```

Project-specific, but we avoid unnecessary line breaks when we can. We start entertaining the idea when a line reaches 80 chars. 120 chars is the hard limit.